.github/workflows/build.yaml

```diff
@@ -121,6 +121,7 @@ jobs:
         platform: [ubuntu20, ubuntu22]
     runs-on: ["self-hosted", "enf-x86-hightier"]
     container: ${{fromJSON(needs.d.outputs.p)[matrix.platform].image}}
+      options: --privileged
```

Comment:
The documentation and example on how to do this is at https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idcontainer

It's something like
```yaml
container:
  image: ${{fromJSON(needs.d.outputs.p)[matrix.platform].image}}
  options: --privileged
```

Reply:
It seems to be accepting this format, then not honoring it.